editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/document_node_types.rs

```diff
@@ -1913,6 +1913,51 @@ fn static_nodes() -> Vec<DocumentNodeType> {
 			properties: node_properties::no_properties,
 			..Default::default()
 		},
+		DocumentNodeType {
+			name: "OR",
+			category: "Logic",
+			identifier: NodeImplementation::proto("graphene_core::logic::LogicOrNode<_>"),
+			inputs: vec![
+				DocumentInputType::value("First", TaggedValue::Bool(false), true),
+				DocumentInputType::value("Second", TaggedValue::Bool(false), true),
+			],
+			outputs: vec![DocumentOutputType::new("Output", FrontendGraphDataType::Boolean)],
+			properties: node_properties::logic_operator_properties,
+			..Default::default()
+		},
+		DocumentNodeType {
+			name: "AND",
+			category: "Logic",
+			identifier: NodeImplementation::proto("graphene_core::logic::LogicAndNode<_>"),
+			inputs: vec![
+				DocumentInputType::value("First", TaggedValue::Bool(false), true),
+				DocumentInputType::value("Second", TaggedValue::Bool(false), true),
+			],
+			outputs: vec![DocumentOutputType::new("Output", FrontendGraphDataType::Boolean)],
+			properties: node_properties::logic_operator_properties,
+			..Default::default()
+		},
+		DocumentNodeType {
+			name: "XOR",
+			category: "Logic",
+			identifier: NodeImplementation::proto("graphene_core::logic::LogicXorNode<_>"),
+			inputs: vec![
+				DocumentInputType::value("First", TaggedValue::Bool(false), true),
+				DocumentInputType::value("Second", TaggedValue::Bool(false), true),
```

Comment:
Thoughts about naming "Second" here @Keavon? "Operator" is used in the properties for this and I'm wondering if we should try and be consistent with the naming?

Reply:
I think maybe "Operand A" and "Operand B" might be the best options? I don't love it but can't think of anything better.

Reply:
Using "Operand B" for the properties panel as well? Or keep it is just "Operator"?

Reply:
I think we'd want to be consistent so it should be called "Operand B" in the Properties panel also. "Operator" would be incorrect (that refers to the AND/OR/XOR/etc., not the left and right hand values which are the operands).

Reply:
I also used that naming for the `Min`, `Max` and `Equality`Nodes, should I update those as well?
(I think I originally got the names from the parameters of the functions that implement the nodes)

Reply:
Yes - I think it is probably best to try and keep the things consistent.